packages/stems/src/components/SelectablePill/SelectablePill.module.css

```diff
@@ -0,0 +1,67 @@
+.pill {
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--text-subdued);
+  user-select: none;
+  height: 24px;
+  padding-left: var(--unit-3);
+  padding-right: var(--unit-3);
+
+  border-radius: 100px;
+  border: 1px solid var(--border-strong);
+  background-color: var(--white);
+
+  font-size: var(--font-m);
+  font-weight: var(--font-medium);
+  line-height: 125%;
+}
+
+.large {
+  height: 32px;
+  box-shadow: var(--shadow-near);
+  color: var(--text-default);
+  padding-left: var(--unit-4);
+  padding-right: var(--unit-4);
+}
+
+.selected,
+.pill:active {
+  background-color: var(--secondary-dark-1);
+  border: 1px solid var(--secondary-dark-1);
+  color: var(--static-white);
+  transition: all 0.12s ease-out;
+}
+
+.pill:hover {
+  border: 1px solid var(--secondary);
+  background-color: var(--secondary-light-1);
+  color: var(--static-white);
+  transition: all 0.12s ease-out;
+}
+
+.large:hover {
+  border: 1px solid var(--secondary-light-1);
+  background-color: var(--secondary-light-2);
+  box-shadow: none;
+}
+
+.large:active,
+.large.selected {
+  box-shadow: none;
+}
+
+.icon path {
+  fill: currentColor;
+}
+
+.icon {
+  margin-right: var(--unit-1);
```

Comment:
Can I recommend using gap on the container instead of adding margins to icons?

Reply:
yess sg